### ISSUE TITLE
website archipelproject.org dead

### ISSUE THREAD

**Issue by lsascha** [ArchipelProject/Archipel#1212]
unfortunately the project website seems dead. Only partially cached page appears.
sad that this nice project dies like this. :worried:

**Comment by yhaenggi** [comment]
you dont need the website to use archipel. its all in git anyway.

**Comment by lsascha** [comment]
thats true. But since after some apt update etc. archipel stopped working (probably because of ejabber update like mentioned in one of the issues) and vnc connections do not work reliably etc etc. all these issues i have the feeling that this project is dying more and more.

Also the page http://app.archipelproject.org is not working anymore where the GUI was hosted before.
When you don't have the gui installed, you have to do it now.

**Comment by devrob20** [comment]
looks like some hacked their domain or is hosting spam on it be carefull some should update this github info and inform  https://www.linux-kvm.org/page/Management_Tool
